src/js/modules/core/components/MastheadBar.js

```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
 import injectSheet from "react-jss";
 import { Link } from "react-router-dom";
 
+import { FaBars, FaSearch } from "../icons";
```

Comment:
But you're building the SVG yourself.... so why bother using their naming. It's confusing